lib/colors.js

```diff
@@ -53,7 +53,7 @@ colors.disable = function() {
 };
 
 colors.stripColors = colors.strip = function(str) {
-  return ('' + str).replace(/\x1B\[\d+m/g, '');
+  return ('' + str);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/labspl/colors.js/pullRequest?prid=8695913)
```suggestion
  return ("" + str);
```